Mixpanel/Mixpanel.m

```diff
@@ -1667,16 +1667,17 @@ - (void)notificationController:(MPNotificationViewController *)controller wasDis
     };
 
     if (status && controller.notification.callToActionURL) {
-        MixpanelDebug(@"%@ opening URL %@", self, controller.notification.callToActionURL);
-        BOOL success = [[UIApplication sharedApplication] openURL:controller.notification.callToActionURL];
+        [controller hideWithAnimation:NO completion:^{
```

Comment:
Agree. Have changed to dismiss with animation. Also rebased on master and force pushed.